docs/docusaurus/docs/cloud/validations/manage_validations.md

```diff
@@ -24,7 +24,7 @@ When you run a validation on an Expectation, a Checkpoint is added. The Checkpoi
 
 5. When the confirmation message appears, click **See results**, or click the **Validations** tab and select the Validation in the **Batches & run history** pane.
 
-6. Optional. Click **Share** to copy the URL for the Validation Results and share them with another GX Cloud user.
+6. Optional. Click **Share** to copy the URL for the Validation Results and share them with other users in your organization.
```

Comment:
**note for reviewer** I made this more specific to reduce risk of doubt about how secure validation results are.

Reply:
FYI I think will be changing int the future, but there was a feature (the first thing I did when I joined 😄) where if you followed a link to an org your weren't part of, you'd get a button to request access. Not positive if it's been removed yet, but maybe it doesn't matter.